localization/languages/zh-CN.json

```diff
@@ -273,21 +273,21 @@
     "passwordManagerSetupStep2": "然后把工具拖动到下面的框中：",
     "passwordManagerSetupDragBox": "把工具拖动到这里",
     "passwordManagerSetupInstalling": "安装中...",
-    "passwordManagerBitwardenSignIn": null, //missing translation
+    "passwordManagerBitwardenSignIn": "登录你的密码管理器以启用自动填充，你的凭据不会保存在 Min 中",
```

Comment:
I think this the translation for `setupSignIn`; `bitwardenSignIn` should contain a URL:

https://github.com/minbrowser/min/blob/c3b1312b4f0eff64a5e03deb7aacde4e848438a5/localization/languages/en-US.json#L276